### PR TITLE
OSDOCS4709: Release notes for Network Observability Operator

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Version(s): 4.12

Issue: https://issues.redhat.com/browse/OSDOCS-4709

Link to docs preview: https://54099--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#new-network-observability

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by jotak** [comment]
LGTM!

**Review by jotak — APPROVED** [review]

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -405,6 +405,12 @@ With {product-title} 4.12, a new interface has been added to the `nodeip-configu
 
 For more information, see xref:../support/troubleshooting/troubleshooting-network-issues.html#overriding-default-node-ip-selection-logic_troubleshooting-network-issues[Optional: Overriding the default node IP selection logic]. 
 
+[id="new-network-observability"]
+==== New Network Observability Operator to observe network traffic flow
+The cluster administrators can now observe the network traffic for {product-title} clusters by installing the Network Observability Operator. The Network Observability uses the eBPF technology to create network flows. The network flows are enriched with {product-title} information and stored in Loki. As an administrator, you can monitor the network traffic in the {product-title} console for detailed troubleshooting and analysis. This feature helps administrators to get insights from different graphical representations of traffic flow.
+
+//TODO: For more information, see
```
Comment (mariomac) [inline release_notes/ocp-4-12-release-notes.adoc:412]: Is this intended?

Comment (snarayan-redhat) [reply release_notes/ocp-4-12-release-notes.adoc:412]: Yes, I will be placing a link to NO overview page once it is available.

**Review by mariomac — APPROVED** [review]

File: release_notes/ocp-4-12-release-notes.adoc [hunk 2]
```diff
@@ -405,6 +405,12 @@ With {product-title} 4.12, a new interface has been added to the `nodeip-configu
 
 For more information, see xref:../support/troubleshooting/troubleshooting-network-issues.html#overriding-default-node-ip-selection-logic_troubleshooting-network-issues[Optional: Overriding the default node IP selection logic]. 
 
+[id="new-network-observability"]
+==== New Network Observability Operator to observe network traffic flow
+The cluster administrators can now observe the network traffic for {product-title} clusters by installing the Network Observability Operator. The Network Observability uses the eBPF technology to create network flows. The network flows are enriched with {product-title} information and stored in Loki. As an administrator, you can monitor the network traffic in the {product-title} console for detailed troubleshooting and analysis. This feature helps administrators to get insights from different graphical representations of traffic flow.
```
Comment (stleerh) [inline release_notes/ocp-4-12-release-notes.adoc:410]: Suggest: "A cluster administrator..."  
  
By using the word "clusters" in the first sentence, it seems to imply that Network Observability can manage more than one cluster.  Suggest "can now observe the network traffic for an {product-title} cluster by installing..."  
  
Suggest: "The Network Observability uses eBPF technology".  Remove "the".  
  
Suggest: "This feature helps admiinstrators get insights".  Remove "to".

**Review by jpinsonneau — APPROVED** [review]

**Review by memodi — APPROVED** [review]

File: release_notes/ocp-4-12-release-notes.adoc [hunk 3]
```diff
@@ -581,6 +581,12 @@ With {product-title} 4.12, a new interface has been added to the `nodeip-configu
 
 For more information, see xref:../support/troubleshooting/troubleshooting-network-issues.html#overriding-default-node-ip-selection-logic_troubleshooting-network-issues[Optional: Overriding the default node IP selection logic]. 
 
+[id="new-network-observability"]
+==== New Network Observability Operator to observe network traffic flow
+As an administrator, you can now install Network Observability Operator to observe the network traffic for {product-title} cluster in the console. You can view and monitor the network traffic data in different graphical representations. The Network Observability uses eBPF technology to create the network flows. The network flows are enriched with {product-title} information, and stored in Loki. As an administrator, you can use the network traffic information for detailed troubleshooting and analysis.
```
Comment (skrthomas) [inline release_notes/ocp-4-12-release-notes.adoc:586]:
```suggestion
As an administrator, you can now install the Network Observability Operator to observe the network traffic for {product-title} cluster in the console. You can view and monitor the network traffic data in different graphical representations. The Network Observability Operator uses eBPF technology to create the network flows. The network flows are enriched with {product-title} information, and stored in Loki. You can use the network traffic information for detailed troubleshooting and analysis.
```

Comment (skrthomas) [reply release_notes/ocp-4-12-release-notes.adoc:586]: A couple of minor things, adding "the" and "Operator". I'm thinking, delete that 2nd "As an administrator".

Comment (snarayan-redhat) [reply release_notes/ocp-4-12-release-notes.adoc:586]: Yes, agree on removing the 'As an administrator..'

**Pull request merged**